### PR TITLE
Transaction payment runtime apis: query call info and fee details

### PR DESCRIPTION
**Pull request by muharem**
Fixes https://github.com/paritytech/substrate/issues/11665

New runtime api introduced in PR https://github.com/paritytech/substrate/pull/11819

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -423,7 +423,7 @@ dependencies = [
 [[package]]
```
Comment (muharem) [inline Cargo.lock:423]: result of `cargo update -p pallet-transaction-payment-rpc-runtime-api`

**Review by athei — APPROVED** [review]

**Comment by muharem** [comment]
`continuous-integration/gitlab-check-runtime` check fail can be ignored, explained - https://github.com/paritytech/ci_cd/issues/538

**Review by bkchr — APPROVED** [review]

**Review by kianenigma — APPROVED** [review]

**Comment by bkchr** [comment]
@muharem you now need to merge master and then this is mergable.

**Comment by muharem** [comment]
bot merge

**Pull request merged**